src/prpy/planning/vectorfield.py

```diff
@@ -558,7 +559,9 @@ def fn_callback(t, q):
                 nonlocals['exception'] = e
                 return -1  # Stop.
 
-        with self.robot_checker_factory(robot) as robot_checker:
+        with self.robot_checker_factory(robot) as robot_checker, \
+            robot.CreateRobotStateSaver(Robot.SaveParameters.ActiveDOF |
+                            Robot.SaveParameters.LinkTransformation):
```

Comment:
I don't think we need `ActiveDOF` here, since we do not alter the robot's active DOF indices.